prow/kube/prowjob.go

```diff
@@ -68,11 +70,12 @@ const (
 )
 
 type ProwJob struct {
-	APIVersion string        `json:"apiVersion,omitempty"`
-	Kind       string        `json:"kind,omitempty"`
-	ObjectMeta ObjectMeta    `json:"metadata,omitempty"`
-	Spec       ProwJobSpec   `json:"spec,omitempty"`
-	Status     ProwJobStatus `json:"status,omitempty"`
+	APIVersion        string `json:"apiVersion,omitempty"`
```

Comment:
I am leaving type meta as is for now. I will change to metav1.TypeMeta once we switch to use client-go (otherwise, it won't work).